src/rgw/rgw_rados.cc

```diff
@@ -3115,7 +3115,7 @@ class RGWDataNotifier : public RGWRadosThread {
   RGWDataNotifierManager notify_mgr;
 
   uint64_t interval_msec() override {
-    return cct->_conf->rgw_md_notify_interval_msec;
+    return cct->_conf->rgw_data_notify_interval_msec;
```

Comment:
if this doesn't exist in `legacy_config_opts.h`, we need to access it by string instead. something like this?
```c++
  return cct->_conf->get_val<int>("rgw_data_notify_interval_msec");
```